src/Components/ProjectHeader/index.js

```diff
@@ -8,15 +8,15 @@ import {
   headerLink,
 } from './style.module.css';
 
-const ProjectHeader = ({artistName, title, materials, link}) => (
-  <section className={container}>
+const ProjectHeader = ({artistName, title, materials, link, children}) => (
+  <div className={container}>
     <h3 className={headerArtist}>{artistName}</h3>
     <h2 className={headerTitle}>{title}</h2>
-    <div className={headerMaterials}>{materials}</div>
+    <div className={headerMaterials} dangerouslySetInnerHTML={{__html: materials}} />
```

Comment:
My favorite function :{ should be fine though cause there's no opportunity for XSS

Reply:
haha, it really is the best. But yeah this is a safe situation for it.